pnpm-lock.yaml

```diff
@@ -3057,7 +3057,7 @@ packages:
     resolution: {integrity: sha512-I/lGHztkCzvwlXpjD2+SNmvNQvB4227xBXhISPjEaJUXGImOQ9f3D2Yj/T3KasSI/h0MLWy74X0J6clhPmsRbQ==}
     engines: {node: '>=12.0.0'}
     peerDependencies:
-      '@types/react': ^17.0.0 || ^18.0.0
+      '@types/react': 18.3.3
       react: ^17.0.0 || ^18.0.0
```

Comment:
Quite strange. Could it be a modification from another work that leaked in this PR?

Reply:
Result of `pnpm dedupe` 🤔 
happens on master as well

I'm not sure why the ci check `pnpm dedupe --check` doesn't pick it up, it also doesn't pick up locally. I guess because it is a peer dep? 🤔

Reply:
I ran dedupe to try to fix the netlify build, just in case 😓